plenum/config.py

```diff
@@ -139,10 +137,10 @@
 # This parameter defines minimal count of accumulated latencies for each client
 MIN_LATENCY_COUNT = 10
 
-latency_averaging_strategy_class = MedianHighStrategy
-throughput_averaging_strategy_class = MedianLowStrategy
+latency_averaging_strategy_class_name = 'MedianHighStrategy'
```

Comment:
Maybe we can just move the strategies to separate files from monitor.py, so that monitor.py's pluginManager  is not called?

Reply:
Also we may consider having enum of strategies and a factory method for creation of a strategy.